### ISSUE TITLE
🐛 [BUG]: Covering a node with handle does not allow the area to be used as a handle

### ISSUE THREAD

**Issue by heaven-born** [bcakmakoglu/vue-flow#990]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

`connectionEndHandle` is always undefined.
Looks like the issue was introduced in version 1.19.3 because in 1.19.2 it is working. 

I can't reproduce it on `Vue Flow: Basic` sandbox, because when I run 'npm install' to update vue flow to the new version the whole server crashes, but on my local env `connectionEndHandle` is always undefined, so I can't 'watch' it. 

### Expected Behavior

expect to see some value inside `connectionEndHandle`

### Steps To Reproduce

Just try to 'watch'  `connectionEndHandle` and you will notice that there wont be any events. won't

**Comment by bcakmakoglu** [comment]
I can't reproduce this issue, works fine for me on `1.20.1` 

Output of a watcher for `connectionEndHandle`:
<img width="618" alt="image" src="https://github.com/bcakmakoglu/vue-flow/assets/78412429/910c269d-09b9-4654-a046-06e5f66f98a6">

**Comment by bcakmakoglu** [comment]
You realize this property will only be set and not `null` when you draw a *Connection Line* and you're currently over a handle with your mouse.
Otherwise there is no end handle to check 😉

**Comment by bcakmakoglu** [comment]
[Here's a working sandbox using `1.20.1`.](https://codesandbox.io/p/sandbox/zen-golick-r3p95v?file=%2Fsrc%2FApp.vue%3A36%2C1-43%2C15)

**Comment by bcakmakoglu** [comment]
Draw a connection from one handle to another and you'll see that the connection end handle info will popup in the panel

**Comment by heaven-born** [comment]
yeah. the `connectionEndHandle` is working. Looks like this is a different kind of issue. 
I stretched my handle to 100% height/width of my node and expected `connectionEndHandle` to be set to not null when I draw the line to this area. And it worked like this in 1.19.2 , but now it only reacts on the original location of the handle and connection radius. Stretching the handle doesn't give much effect anymore. 

So now I ended up in a situation where I don't understand how to make my whole node area behave as a handle. The connection radius doesn't fit the shape of my node, so I can't increase it.  

Any ideas on how to cover the whole node area with the handle? In 1.19.2  I could drop my connection line in any place of the stretched handle and it was enough for the connection.
